build_and_release.sh

```diff
@@ -18,7 +18,10 @@ platforms=(
   windows-arm64
 )
 
-if [[ $GITHUB_REF = refs/tags/* ]]; then
+if [ -n "$GH_RELEASE_TAG" ]; then
+  echo "invoking release tag override $GH_RELEASE_TAG"
```

Comment:
Not sure this output is necessary.

Reply:
Removed in [8e1e8c2](https://github.com/cli/gh-extension-precompile/pull/37/commits/8e1e8c21c5e04c619f746fab9d199d59faef4ccd)